cli/src/config/templates.toml

```diff
@@ -365,11 +365,15 @@ time_range.end().ago() ++ label("time", ", lasted ") ++ time_range.duration()
 'format_commit_summary_with_refs(commit, refs)' = '''
 label(if(commit.current_working_copy(), "working_copy"),
   separate(" ",
-    format_short_change_id_with_hidden_and_divergent_info(commit),
+    format_short_change_id_with_change_offset(commit),
     format_short_commit_id(commit.commit_id()),
     separate(commit_summary_separator,
       refs,
       separate(" ",
+        coalesce(
+          if(commit.hidden(), label("hidden", "(hidden)")),
+          if(commit.divergent(), label("divergent", "(divergent)")),
+        ),
         if(commit.conflict(), label("conflict", "(conflict)")),
```

Comment:
Wouldn't it be better to use either `(hidden)` or `hidden` form thoroughly? I would use `(hidden)` form because other labels like `(empty)` are all parenthesized.